app/models/video.rb

```diff
@@ -12,4 +12,12 @@ def self.search_by_title(search_term)
     return [] if search_term.blank?
     where("title LIKE ?", "%#{search_term}%").order("created_at DESC")
   end
+
+  def rating
+    ratings = reviews.map(&:rating)
```

Comment:
Why you don't you just use `average` method from ActiveRecord? Its more succinct. http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average